### PR TITLE
Vivek_Assignment_14

### PR DESCRIPTION
**Pull request by vivekvitthalpatil**
image

image

image

### CODE REVIEW

File: src/com/Technocredits/orghrm/base/PredefinedActions.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package com.Technocredits.orghrm.base;
```
Comment (harshvegada) [inline src/com/Technocredits/orghrm/base/PredefinedActions.java]: the package name should be in small letters.

File: src/com/Technocredits/orghrm/base/TestBase.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+	public void setup() {
+		System.out.println("Launch the OrgHRM application");
+		PredefinedActions.start();
+		LoginPage loginpage = new LoginPage();
+		loginpage.enterCredentials("Admin", "AXrKSz@24g");
+		loginpage.clickOnLoginButton();
+	}
+	
+		public void teardown() {
```
Comment (harshvegada) [inline src/com/Technocredits/orghrm/base/TestBase.java]: code formatting require.

**Review by harshvegada — APPROVED** [review]
code looks good to me, a small comment.
kindly address & merge it.

**Comment by harshvegada** [comment]
delete package 1 from code base if not require

**Comment by vivekvitthalpatil** [comment]
@harshvegada Code formatting and package naming issue are resolved.

**Comment by harshvegada** [comment]
Thanks @vivekvitthalpatil

**Pull request merged**